common/src/main/java/org/apache/uniffle/common/config/RssClientConf.java

```diff
@@ -209,4 +209,10 @@ public class RssClientConf {
           .defaultValue(false)
           .withDescription(
               "Whether to support task write failed retry internal, default value is false.");
+
+  public static final ConfigOption<Integer> RSS_CLIENT_REGISTER_SHUFFLE_MAX_PARALLELISM =
+      ConfigOptions.key("rss.client.registerShuffle.maxParallelism")
```

Comment:
Our config options aren't good enough. Some config options have bad name. Maybe we should refactor them.
I thinks we should use `rss.client.group.camelStyleConfigOption`. I prefer using `rss.client.rpc.maxRegisterParallelism`. WDYT?

Reply:
+1.